File: cpp/wej-wyj.cpp
/*
 * wej-wyj.cpp
 */

#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    const int rozmiar = 25;
    char a[rozmiar];
    char b[rozmiar];
	cout << "Podaj imię: ";
    cin.getline(a, rozmiar);
	cout << "Podaj nazwisko: ";
    cin.getline(b, rozmiar);
    
    cout << endl << "WITAJ: " << a << " " << b;
    
	return 0;
}

File: python/wej-wyj.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    a = input("Podaj imię: ")
    b = input("Podaj nazwisko: ")
    
    print("WITAJ:", a, b)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
